projects/ngrx.io/content/guide/component-store/effect.md

```diff
@@ -91,3 +91,23 @@ An easy way to handle the response in ComponentStore effects in a safe way, with
     );
   });
 </code-example>
+
+## Calling an `effect` without parameters
+
+A common use case is to call the `effect` method without any parameters. 
+To make this possible:
+- The generic type of the `effect` method needs to be set to `void`.
+- The name of the source stream doesn't matter: `trigger$`, `source$` or `$` are good names.
+
+<code-example header="movies.store.ts">
+  readonly getAllMovies = this.effect&lt;void&gt;(
```

Comment:
Should we expand this example, to also include how it can be invoked?

Reply:
Yes you are write, but that's can be in another MR. One thing that should be added as well in the doc is the fact you can call an effect and updater with an observable. I think it's missing.